android/src/fdroid/play/release-notes/de-DE/default.txt

```diff
@@ -1,9 +1,10 @@
-• neue Kartendaten vom 16. August
-• längere Routen werden nun deutlich schneller berechnet
-• die Suchfunktion ist nun zuverlässiger und schneller
-• reduzierter Speicherplatzverbrauch der Offlinekarten
-• Fehler, dass manche Suchergebnisse nicht auf der Karte angezeigt wurden, wurde behoben
-• verbesserte Darstellung von Militärgebieten und Grenzen
-• barrierefreie Parkplätze werden nun auf der Karte angezeigt
-• verschiedene Icons für Sportarten hinzugefügt
-meht unter omaps.app/news
+• Neue OSM-Kartendaten ab dem 12. September
+• Entfernte untere Symbolleiste für bessere UX und Sichtbarkeit der Karte
```

Comment:
"Entfernte untere Symbolleiste ..." sounds somehow wrong for me. Use "Untere Symbolleiste entfernt ..." instead.